frontend/src/containers/members/MenuMember.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, HStack } from "@chakra-ui/react";
+
+const MenuMember = () => {
+  return (
+    <Box
+      position={"fixed"}
+      right={"0"}
+      left={"0"}
+      bottom={"0"}
+      backgroundColor={"white"}
+      py={"4"}
+      display={"flex"}
+      justifyContent={"center"}
+    >
+      <HStack
+        justifyContent={"space-between"}
+        mx={"4"}
+        border={"1px"}
+        borderRadius={"16px"}
+        padding={"1"}
+        backgroundColor={"gray.700"}
+        position={"relative"}
+        maxW={"440px"}
+        w={"full"}
+        py={"1"}
+      >
+        <Box
+          w={"50%"}
+          textAlign={"center"}
+          borderRadius={"12px"}
+          py={"4"}
+          backgroundColor={"gray.100"}
+        >
+          Members
+        </Box>
+        <Box
+          w={"50%"}
+          textAlign={"center"}
+          borderRadius={"16px"}
+          py={"4"}
+          color={"white"}
+        >
+          Bills
+        </Box>
```

Comment:
In my opinion, kayanya background color bisa ditentukan secara dynamic sesuai state yang aktif yang mana deh? 

Jadinya, kalau Members nya yang lagi diklik dia bakal berubah background color nya, begitu juga ketika si Bills nya diklik. Wdyt?

Reply:
untuk sekarang masih belum mikirin apakah ada state apa yang harus digunakan pada element, mungkin next update bakal ada di update ketika semua menu sudah selesai

Reply:
Kalau mau even better, sebenernya bisa pake semantic aja (e.g. "primary", "secondary" dll). Nanti bisa setup theme aja : https://chakra-ui.com/docs/styled-system/semantic-tokens#theme-example

Jadi nanti kita bisa ganti "primary" color dari app kita

Reply:
Bisa berdasarkan props aja dan, gak perlu state, biarkan component ini jadi dummy aja. Manage statenya dari luar aja

Reply:
Nanti props nya tinggal ditentuin di masing2 page : 

```
// Billing Page : 
<Layout>
  ...
  <BottomMenu activedMenu="billings" />
</Layout>

// Members Page : 
<Layout>
  ...
  <BottomMenu activedMenu="members" />
</Layout>
```

Reply:
wdyt @dannycahyo @AndriP8 ?

Reply:
Agree, lewat props pun ga masalah sih. Concernku sebenarnya di styling background color menu nya. 

Jadinya kita bisa tau, Menu bagian mana yang lagi aktif atau dikunjungi.

Reply:
yupp bener, jadinya ntar bakal ada ada kondisi seperti ini hanya untuk mengetahui menu mana yang lagi aktif 
```
activeMenu === "billings" ? 
<Box
          w={"50%"}
          textAlign={"center"}
          borderRadius={"12px"}
          py={"4"}
          backgroundColor={"gray.100"}
        >
          Members
        </Box>
:
<Box
          w={"50%"}
          textAlign={"center"}
          borderRadius={"12px"}
          py={"4"}
          backgroundColor={"gray.700"}
        >
          Members
        </Box>
````

Reply:
Solve in sync discussion => Baiknya ga perlu maintain banyak id